### ISSUE TITLE
`Dockerfile`-less build issues

### ISSUE THREAD

**Issue by weikanglim** [Azure/azure-dev#4575]
This issue tracks all issues that would prevent adoption of `Dockerfile`-less Oryx builds.

- [ ] https://github.com/Azure/azure-dev/issues/3124
- [ ] https://github.com/microsoft/Oryx/issues/2157 
- [ ] https://github.com/microsoft/Oryx/issues/2525

**Comment by spboyer** [comment]
@weikanglim @kristenwomack  - we should close this and related issues as Oryx++ is not moving forward on this feature. Perhaps we move to suggesting to "Add Dockerfile" via Copilot if the host is containerapps, aks.

**Comment by rajeshkamal5050** [comment]
> [@weikanglim](https://github.com/weikanglim) [@kristenwomack](https://github.com/kristenwomack) - we should close this and related issues as Oryx++ is not moving forward on this feature. Perhaps we move to suggesting to "Add Dockerfile" via Copilot if the host is containerapps, aks.

Agreed. For any new issues coming on simplified-init/easy-init -> Oryx/++ we can recommend as above.
